project/resources/factories.py

```diff
@@ -1 +1,40 @@
-# TODO
+import uuid
+from random import choice
+
+from factory import DjangoModelFactory, Faker, LazyAttribute, SubFactory
+
+from users.factories import UserFactory
+from .models import Resource
+
+
+RESOURCE_TYPES = [
+    ('VID', 'Video'),
+    ('POD', 'Podcast'),
+    ('PODEP', 'Podcast Episode'),
+    ('TALK', 'Talk'),
+    ('TUTOR', 'Tutorial'),
+    ('COURSE', 'Course'),
+    ('BOOK', 'Book'),
+    ('BLOG', 'Blog'),
+    ('GAME', 'Game'),
+    ('EVENT', 'Event'),
+    ('TOOL', 'Tool'),
+    ('LIB', 'Library'),
+    ('WEB', 'Website')
+]
```

Comment:
There are a couple of issues here:
First is we want to pass `choice()` a straight list, not a list of tuples (unlike django's `choices` which accepts a list of tuples).
Second is we don't really want to declare and maintain this list in two places. If we change it in the model we want our factory to update too

Lets use a list comprehension and declare `RESOURCE_TYPES` as:

```py
RESOURCE_TYPES = [
    r[0] for r in Resource.RESOURCE_TYPES
]
```

That would would solve both of those problems.

Reply:
@chris48s - excellent suggestion.  I only have one quibble here (and yes, I am being a cranky-pants dev....BUT) single-letter variable names make me _squirm_.  They almost never help readability..especially in a comprehension.  Would be much happier with: 

```python
RESOURCE_TYPES = [resource[0] for resource in Resource.RESOURCE_TYPES] 
```

...or something along those lines.  😉